src/main/kotlin/org/rust/lang/core/macros/MacroExpander.kt

```diff
@@ -346,7 +346,7 @@ class MacroExpander(val project: Project) {
     }
 
     companion object {
-        const val EXPANDER_VERSION = 13
+        const val EXPANDER_VERSION = 14
```

Comment:
Let's increment a version in MacroExpansionSharedCache (add `+ 1`) instead of modifying this version that also affects other stuff